.github/workflows/dependabot-prs.yml

```diff
@@ -14,3 +14,9 @@ jobs:
         alert-lookup: true
         compat-lookup: true
         github-token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Add a label for all PRs with an alert state
+      if: ${{ steps.dependabot-metadata.outputs.alert-state != '' }}
```

Comment:
From [this](https://docs.github.com/en/actions/learn-github-actions/expressions#functions) I understand Null is captured as '' in GH actions but not sure...